doc/tutorials/Chapter_1/01-Setup_Installation.md

```diff
@@ -0,0 +1,302 @@
+  Simulator.registerAction(100, () {
+    if (!noPrint) {
+      print('Simulation completed!');
+    }
+  });
+
+  // Set a maximum time for the simulation so it doesn't keep running forever.
+  Simulator.setMaxSimTime(100);
+
+  // Kick off the simulation.
+  await Simulator.run();
+
+  // We can take a look at the waves now.
+  if (!noPrint) {
+    print('To view waves, check out waves.vcd with a waveform viewer'
+        ' (e.g. `gtkwave waves.vcd`).');
+  }
+}
+```
+
+Finally, run the main function and observe if you can see the system verilog code generated by running the command:
+
+```shell
+dart run
+```
+
+Well done! Your setup has been completed successfully. The successful generation of system verilog code confirms that your configuration is in good order.
+
+
+### Option 2: Install from Source
+
+Clone ROHD repository to the local directory. (Install From Source). On your terminal, run 
+
+```shell
+cd C:\
+git clone https://github.com/intel/rohd.git
+```
+
+Next, open up your repository in VSCode using the command:
+
+```shell
+code rohd
+```
+
+You will see VSCode automatically open up your ROHD folder. 
+
+![step 2](assets/localSetup/step2.PNG)
+
+Open up terminal in your VSCode by go to view -> terminal. Then, get the rohd package downloaded using the command below.
+
+```cmd
+dart pub get
+```
+Then, run Rohd example.
+
+Open up `example` folder on the left navigation panel and click on `example.dart` to bring forward the first example of ROHD. After that, navigate to the main function at below of line 58 and click on the `Run` at `Run | Debug`.
+
+![step 3](assets/localSetup/step4.PNG)
+
+If you can see SystemVerilog code pop up on the terminal. Congratulation, you are ready with ROHD development! 
+
+
+## Docker Container Setup
+
+There are another options which is to setup on lcoal docker environment. Firstly, you must install docker in your PC.
+
+Pre-requisites:
+
+- Install docker 
+    - https://docs.docker.com/get-docker
+
+Then, you need to clone rohd repository to your local PC. 
+
+```shell
+cd C:\
+git clone https://github.com/intel/rohd.git
+cd rohd
+```
+
+After clone, bring up the VSCode by run this command on your terminal:
+
+```shell
+code .
+```
+
+At this phase, assume that you already install docker on your pc. You can install docker vscode extension.
+
+![vscode extension](assets/docker/vscode_extension.PNG)
+
+Once the extension installed. You might have to refresh your repository by restart vscode and you will see a pop up on the bottom corner right. Click on the little notification icon and clone in volume.
+
+![vscode extension](assets/docker/clone_in_volume.png)
+
+When the setting finished, you can bring up the terminal by navigate to top navigation panel **view -> terminal**.
+
+In the terminal, run:
+
+```shell
+dart example/example.dart
+```
+
+If you can see SystemVerilog code pop up on the terminal. Congratulation, you are ready with ROHD development! 
+
+----------------
+2023 February 13
```

Comment:
Add two spaces at the end of the date line to force a new-line before the author line

Reply:
Okay, btw do you think its necessary to have author lines on every files?

Reply:
It is not necessary, but you may add it if you like